src/theme/components/headerSearchForm.ts

```diff
@@ -0,0 +1,77 @@
+const focusStyle = {
+  borderRadius: "none",
+  outlineColor: "#135772 !important",
+  outlineOffset: "0 !important",
+  outlineStyle: "solid !important",
+  outlineWidth: "0.1875em !important",
+};
+const HeaderSearchForm = {
+  parts: ["desktopSearchBtn", "form", "textInput", "radio"],
+  baseStyle: ({ isMobile }) => ({
+    whiteSpace: "initial",
+    position: "absolute",
+    left: "0px",
+    backgroundColor: "#1B7FA7",
+    width: "100%",
+    zIndex: "99999",
```

Comment:
In the browser, the vertical alignment for the form wrapper is not correct.  It’s too high and covering the menu.  But I don’t see the `top` attribute here.  Where is that `222px` value coming from?

![image](https://user-images.githubusercontent.com/5241828/173689506-8ed71b64-b33d-4869-839a-fcc11fa79042.png)

Reply:
Where are you seeing `222px`? I don't see that in the browser or in the repo. I actually don't see the menu floating too high at all.

Reply:
Weird.  Now it looks fine and I don’t see the `top` attribute applied.